src/include/distributed/connection_cache.h

```diff
@@ -52,26 +52,11 @@ typedef struct NodeConnectionEntry
 } NodeConnectionEntry;
 
 
-/* describes what kind of modifications have occurred in the current transaction */
-typedef enum
-{
-	XACT_MODIFICATION_INVALID = 0, /* placeholder initial value */
-	XACT_MODIFICATION_NONE,        /* no modifications have taken place */
-	XACT_MODIFICATION_DATA,        /* data modifications (DML) have occurred */
-	XACT_MODIFICATION_SCHEMA       /* schema modifications (DDL) have occurred */
-} XactModificationType;
-
-
-/* state needed to prevent new connections during modifying transactions */
-extern XactModificationType XactModificationLevel;
-
-
 /* function declarations for obtaining and using a connection */
 extern PGconn * GetOrEstablishConnection(char *nodeName, int32 nodePort);
 extern void PurgeConnection(PGconn *connection);
 extern void BuildKeyForConnection(PGconn *connection, NodeConnectionKey *connectionKey);
 extern PGconn * PurgeConnectionByKey(NodeConnectionKey *nodeConnectionKey);
-extern bool SqlStateMatchesCategory(char *sqlStateString, int category);
 extern void WarnRemoteError(PGconn *connection, PGresult *result);
 extern void ReraiseRemoteError(PGconn *connection, PGresult *result);
```

Comment:
Which of these? You mean WarnRemoteError/ReraiseRemoteError? There's a variant of them in remote_commands, but it's a slightly different API. Partially because when I wrote that code we still purged connections upon errors...